apdbPlots.py

```diff
@@ -532,20 +528,17 @@ def plotSeeingHistogram(repo, sourceTable, ccd=35, ccdXcenter=1000,
         miniCalexp = butler.get('calexp_sub', visit=int(visit),
                                 ccd=ccd, bbox=lsst.geom.Box2I())
         psf = miniCalexp.getPsf()
-        shape = psf.computeShape(lsst.geom.Point2D(ccdXcenter, ccdYcenter))
-        radii.append(shape.getDeterminantRadius()*2.355)  # convert sigma to FWHM
+        psfSize = psf.computeShape().getDeterminantRadius()
+        radii.append(psfSize*2.355)  # convert sigma to FWHM
```

Comment:
Does the x, y dependence not matter?

Reply:
I'm 99% sure it doesn't, i.e., the PSF is defined for the whole image.